cpp/tests/utilities/column_wrapper.cuh

```diff
@@ -525,6 +525,56 @@ struct column_wrapper {
 
   gdf_size_type size() const { return the_column.size; }
 
+  /**---------------------------------------------------------------------------*
+   * @brief Prints the values of the underlying gdf_column to a string
+   * 
+   *---------------------------------------------------------------------------**/
+  std::string to_str() const {
+    std::ostringstream buffer;
+    print_gdf_column(&the_column, 1, buffer);
+    return buffer.str();
+  }
+
+  /**---------------------------------------------------------------------------*
+   * @brief Functor for comparing if two elements between two gdf_columns are
+   * equal.
+   *
+   *---------------------------------------------------------------------------**/
+  struct elements_equal {
```

Comment:
I think this was mistakenly added back during a merge after it was removed in a previous commit.